cg/apps/sequencing_metrics_parser/models/bcl_convert.py

```diff
@@ -0,0 +1,92 @@
+from pydantic import BaseModel, BaseConfig, Field
+import xml.etree.ElementTree as ET
```

Comment:
In a future PR this should move to the IO module

Reply:
Yes, I will make it the next one!

There is a library that extends the xml parsing:
import defusedxml.defuse_stdlib
defusedxml.defuse_stdlib()
# Now you can use xml.etree.ElementTree as usual
import xml.etree.ElementTree as ET
tree = ET.parse('your_file.xml')
Might be good to keep in mind for when making the xml io controller